person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, id, name: 'Unknown', parent_permission: true)
+    @age = age
```

Comment:
- [ ] A constructor is a special method in Ruby that gets invoked when an object of this class is created. All arguments pass to _new_  will automatically pass to method initialize. Only three parameter is require to create a person object hence let's kindly remove the id parameter. The person id should be generated automatically.

Reply:
@FrederickMih are you saying , the ID will be generated automatically or i should generate the ID manually in the class

Reply:
Thanks , I understand now, thanks for your suggestion. I think i will have to define a method in order to generate  the id manually